projects/recipe-page/phase-2-css-prompt.md

```diff
@@ -65,14 +66,12 @@ You'll be building on the HTML code you completed in Phase 1 of this project.
 
 #### Part 1 - Create a CSS file in the proper directory of your file tree
 
-1. Using the Terminal, ensure you are in the `recipe-page` directory. Create a file called `styles.css`. You can verify this worked by running the `ls` command.
-2. Use the Terminal shortcut `code .` to open the `recipe-page` directory in VSCode.
-3. Run `git status`. You should see that 'styles.css' is not being tracked yet.
-4. Add and commit, making sure to write a descriptive commit message in the present tense, such as "Create stylesheet."
+1. On your desktop, create a folder called `techtonica-projects`.  Move your `recipe-page.html` file into `techtonica-projects`.  If you no longer have it, make it again with the Challenge directions from [Phase 1](./phase-1-html-prompt.md).
```

Comment:
They already created the folder in part 1, so maybe add the "if you have not already created the folder" part to this

Reply:
The first part was done on replit, I am not sure why we are moving it locally...

Reply:
In the HTML part, we built it on CodePen to start out simple, and then moved it to local at the end.  I figured they would miss out on connecting things through style and script tags if we didn't move to local, but I didn't want  them to get frustrated before writing their first HTML tags with setup overhead. Let me know if you think I should switch to just one or the other.

Reply:
I've made edit's so that this is the first time we're asking them to make these directories.